### DISCUSSION TITLE
Solo Gen 4 need assist

### DISCUSSION THREAD

**Discussion by MarkoKurtovic** [geoffreybennett/alsa-scarlett-gui#94] [General]
Hi guys, if somebody can assist with Gen 4 Solo, for linux ubuntu.

**Comment by geoffreybennett** [comment]
Hi Marko, assist how? The 4th Gen devices are not yet supported by the released version of `alsa-scarlett-gui`, but this is not needed in order to use your interface. Please start with the [FAQ](https://github.com/geoffreybennett/alsa-scarlett-gui/blob/master/FAQ.md) though; that has some info that might explain things for you.

**Reply by MarkoKurtovic** [reply]
yes for gui, I didn't maybe post this at correct project.
I was following command:
options snd_usb_audio vid=0x1235 pid=0x8210 device_setup=1 > /etc/modprobe.d/snd_usb_audio.conf

I tried also sledgehammer, not working.

and I can't find PID for Gen4 solo.

Issue I have is mic is not recognized in sound and devices in linux, output is recognized.

Not sure what to do, do you have any advice how to proceed?

**Comment by geoffreybennett** [comment]
> yes for gui, I didn't maybe post this at correct project.

Correct, this project does not change how the device is recognised.

> I was following command: options snd_usb_audio vid=0x1235 pid=0x8210 device_setup=1 > /etc/modprobe.d/snd_usb_audio.conf

That configuration is only for the 2nd/3rd Gen Scarlett driver, before Linux 6.7.

> I tried also sledgehammer, not working.

That won't do anything. 4th Gen driver doesn't need any options to enable it.

> and I can't find PID for Gen4 solo.

No need for this.

> Issue I have is mic is not recognized in sound and devices in linux, output is recognized.

Probably you need to use `pavucontrol` to set the profile under the configuration tab. "Pro Audio" should be there as an option; that's the one I recommend.

What version of Ubuntu are you using?
